### DISCUSSION TITLE
Call .net from my website.

### DISCUSSION THREAD

**Discussion by eewann** [Voltstro-Studios/UnityWebBrowser#81] [Q&A] [answered]
Hi, i want to call some functions from my website. How can I do that?
For example, i want to instantiate an object when the user clicks a button on my website.

**Comment by Voltstro** [comment, marked as answer]
No, thats not currently possible.

**Reply by eewann** [reply]
Ok :sob:. Hope it is possible very soon.

**Reply by Konijima** [reply]
@Voltstro is it be possible to communicate both way from an embedded page?

**Reply by Voltstro** [reply]
The only form communication that can occur is with JS, and that can currently only be executed via the Unity side with the `ExecuteJS` method.

Having custom JS functions that you setup that will invoke native .NET code is not planned to be worked on until v2.1.
